Content.Server/Cloning/CloningSystem.cs

```diff
@@ -33,431 +30,331 @@
-using Content.Server.EntityList;
-using Content.Shared.SSDIndicator;
-using Content.Shared.Damage.ForceSay;
-using Content.Server.Polymorph.Components;
-using Content.Shared.Chat;
-using Content.Shared.Abilities.Psionics;
-
-namespace Content.Server.Cloning
+using Content.Shared.Random.Helpers;
+using Content.Shared.Contests;
+using Robust.Shared.Serialization.Manager;
+using Robust.Shared.Utility;
+using Timer = Robust.Shared.Timing.Timer;
+using Content.Server.Power.Components;
+using Content.Shared.Drunk;
+using Content.Shared.Nutrition.EntitySystems;
+
+namespace Content.Server.Cloning;
+
+public sealed partial class CloningSystem : EntitySystem
 {
-    public sealed class CloningSystem : EntitySystem
+    [Dependency] private readonly DeviceLinkSystem _signalSystem = default!;
+    [Dependency] private readonly IPlayerManager _playerManager = null!;
+    [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
+    [Dependency] private readonly EuiManager _euiManager = null!;
+    [Dependency] private readonly CloningConsoleSystem _cloningConsoleSystem = default!;
+    [Dependency] private readonly HumanoidAppearanceSystem _humanoidSystem = default!;
+    [Dependency] private readonly ContainerSystem _containerSystem = default!;
+    [Dependency] private readonly MobStateSystem _mobStateSystem = default!;
+    [Dependency] private readonly PowerReceiverSystem _powerReceiverSystem = default!;
+    [Dependency] private readonly IRobustRandom _random = default!;
+    [Dependency] private readonly AtmosphereSystem _atmosphereSystem = default!;
+    [Dependency] private readonly TransformSystem _transformSystem = default!;
+    [Dependency] private readonly SharedAppearanceSystem _appearance = default!;
+    [Dependency] private readonly PuddleSystem _puddleSystem = default!;
+    [Dependency] private readonly ChatSystem _chatSystem = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+    [Dependency] private readonly IConfigurationManager _config = default!;
+    [Dependency] private readonly MaterialStorageSystem _material = default!;
+    [Dependency] private readonly PopupSystem _popupSystem = default!;
+    [Dependency] private readonly SharedMindSystem _mindSystem = default!;
+    [Dependency] private readonly MetaDataSystem _metaSystem = default!;
+    [Dependency] private readonly SharedJobSystem _jobs = default!;
+    [Dependency] private readonly TagSystem _tag = default!;
+    [Dependency] private readonly ContestsSystem _contests = default!;
+    [Dependency] private readonly ISerializationManager _serialization = default!;
+    [Dependency] private readonly DamageableSystem _damageable = default!;
+    [Dependency] private readonly HungerSystem _hunger = default!;
+    [Dependency] private readonly ThirstSystem _thirst = default!;
+    [Dependency] private readonly SharedDrunkSystem _drunk = default!;
+    [Dependency] private readonly MobThresholdSystem _thresholds = default!;
```

Comment:
These are all used???

Reply:
Shockingly yes.

Reply:
![image](https://github.com/user-attachments/assets/da8fa3dd-01ea-45f2-b91d-e153afa6f241)

It horrifies me too because it gives me instant flashbacks to the DeepFryerSystem.